app/src/main/java/de/tum/in/tumcampusapp/managers/CalendarManager.java

```diff
@@ -155,6 +157,28 @@ public Cursor getFromDbForDate(Date date) {
     }
 
     /**
+     * Returns all stored events in the next days from db
+     *
+     * @param dayCount The number of days
+     * @return List<IntegratedCalendarEvent> List of Events
+     */
+    public List<IntegratedCalendarEvent> getNextDaysFromDb(int dayCount) {
+        Calendar calendar = Calendar.getInstance();
+        String from = Utils.getDateTimeString(calendar.getTime());
+        calendar.add(Calendar.DAY_OF_YEAR, dayCount);
+        String to = Utils.getDateTimeString(calendar.getTime());
+
+        List<IntegratedCalendarEvent> calendarEvents = new ArrayList<>();
+        Cursor cursor = db.rawQuery("SELECT * FROM calendar WHERE dtend BETWEEN datetime(?) AND datetime(?) AND status!=\"CANCEL\" ORDER BY dtstart ASC", new String[]{from, to});
```

Comment:
well the ordering is still not working correctly, so i have to do some further work on this query